service/geopm.service

```diff
@@ -6,6 +6,8 @@
 Description=Global Extensible Open Power Manager Service
 StartLimitIntervalSec=0
 StartLimitBurst=3
+Wants=msr-safe.service
```

Comment:
I could see an argument for using "Requires=" here instead of "Wants=", but that would make the msr-safe.service a hard dependency.
The way this is now, if the msr-safe.service is not present, it will *not* block the geopm.service from starting.